### PR TITLE
Get rid of sshd in traffic-manager

### PR DESCRIPTION
**Pull request by thallgren**
## Description

This PR contains the commits needed to get rid of sshd in the traffic-manager.

## Checklist

 - [x] I made sure to update `./CHANGELOG.md`.
 - [x] My change is adequately tested.

### CODE REVIEW

**Comment by LukeShu** [comment]
I think you accidentally included a bunch of rebased commits from https://github.com/telepresenceio/telepresence/pull/1735 when you rebased this.

File: rpc/daemon/daemon.proto [hunk 1]
```diff
@@ -52,5 +53,6 @@ message TableName {
 }
 
 message ManagerInfo {
-  int32 grpc_port = 1;
+  manager.SessionInfo session = 1;
+  int32 grpc_port = 2;
```
Comment (LukeShu) [inline rpc/daemon/daemon.proto]: Probably
```protobuf
  manager.SessionInfo session = 2;
  int32 grpc_port = 1;
  ```
  to avoid re-numbering grpc_port?

**Review by LukeShu — CHANGES_REQUESTED** [review]
Looks good to me, except for the bad rebase.  I pushed `lukeshu/thallgren/no-manager-sshd` that fixes it by merging `release/v2` and then rebasing on to that, if you want to steal that.  Or just drop the commits.

**Comment by thallgren** [comment]
I fixed the MTU in ICMP DNR packages in the TUN-device PR, merged it to release/v2 and rebased this PR on top of that.

File: rpc/daemon/daemon.proto (hunk 1 above)
Comment (thallgren) [reply rpc/daemon/daemon.proto]: I fixed this with a squash while rebasing.

**Review by LukeShu — APPROVED** [review]

**Pull request merged**